### ISSUE TITLE
Remove the limit for layer height

### ISSUE THREAD

**Issue by julie777** [bambulab/BambuStudio#1719]
**Is your feature request related to a problem? Please describe.**

P1P using .4mm nozzle and Bambu Slicer will not let me set layer height to .3 which is very common for a .4 nozzle. Instead it limits the layer height to .28.

**Describe the solution you'd like**

Allow me to set the layer height that I choose. The print profiles provides show me what BL thinks is a good setting. I get that. However, there are many reasons to choose a different height.

**Describe alternatives you've considered**

**Additional context**

**Comment by MadMax389** [comment]
You can set whatever layer height you want on the quality tab regardless of the user profile selected. The print speed will still be regulated by whatever flow rate setting you have in the filament profile.
![image](https://user-images.githubusercontent.com/96731892/235348694-fb5ea18e-48fa-4b4d-9751-adffc72f070c.png)

**Comment by julie777** [comment]
That is not my experience, but now I that I went back and checked, it must have been added in Orca.

![image](https://user-images.githubusercontent.com/5090005/235379298-318e9801-6726-4a24-a13f-e69095ad7013.png)

@MadMax389  Thanks for setting me straight.

**Comment by MadMax389** [comment]
Actually, in Orca, you can raise the upper layer height limit in the printer profile.
![image](https://user-images.githubusercontent.com/96731892/235384535-1164ef72-90e2-4568-bc83-6f861b5b0066.png)

**Comment by julie777** [comment]
Well, I just got the error in BS. I have a .4 nozzle and I am using the .4mm profile.
When I tried to change the layer height to .3, this error popped up.

![image](https://user-images.githubusercontent.com/5090005/235523629-256de75b-8db3-49ca-b317-fd49f559f469.png)

It did leave the .3 in the setting though.

I looked at the printer settings and it does have min and max layer height and max is set to .28. So changing it is not a problem. 

The error message needs to be reworded to indicate that the layer height is not between the min and max in the printer profile being used.
